packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -400,6 +400,32 @@ describe('network stubbing', function () {
     })
   })
 
+  context('network handling', function () {
+    context('can intercept against any domain', function () {
+      beforeEach(function () {
+        // reset origin
+        cy.visit('http://localhost:3500/fixtures/generic.html')
+      })
+
+      it('different origin (HTTP)', function () {
+        cy.route2('/foo').as('foo')
+        .then(() => {
+          $.get('http://baz.foobar.com:3501/foo')
+        })
+        .wait('@foo')
+      })
+
+      // @see https://github.com/cypress-io/cypress/issues/8487
+      it('different origin (HTTPS)', function () {
+        cy.route2('/foo', 'somethin').as('foo')
+        .then(() => {
+          $.get('https://bar.foobar.com:3502/foo')
+        })
+        .wait('@foo')
```

Comment:
and the response is available from other HTTPS domains? Like can we confirm that we can spy on the response from https://jsonplaceholder.cypress.io/ request?

Reply:
i have pushed a commit with tests for intercepting HTTPS responses, am happy to report it works as expected :)